server/models/Notification.ts

```diff
@@ -182,6 +184,40 @@ export class Notification extends Model<InferAttributes<Notification>, InferCrea
     return notifications.map(us => us.UserId);
   }
 
+  static async getUnsubscribers(_where: {
+    type?: ActivityClasses | ActivityTypes;
+    CollectiveId?: number | number[];
+    UserId?: number | number[];
+  }) {
+    debug('getUnsubscribers', { _where });
+    const getUsers = notifications => notifications.map(notification => notification.User);
+
+    const include = [{ model: models.User }] as any;
+    const where = { ..._where, active: false } as any;
+
+    const collective = where.CollectiveId && (await models.Collective.findByPk(where.CollectiveId));
+    if (collective) {
+      where.CollectiveId = flatten([where.CollectiveId, collective.ParentCollectiveId, collective.HostCollectiveId]);
+    }
+
+    const classes = keys(pickBy(ActivitiesPerClass, array => array.includes(where.type)));
+    where.type = flatten([where.type, `${where.type}.for.host`, ...classes]);
+
+    const unsubs = await Notification.findAll({
+      where,
+      include,
+    }).then(getUsers);
+
+    const subs = collective
+      ? await Notification.findAll({
+          where: { ...where, active: true, CollectiveId: [collective.id, collective.ParentCollectiveId] },
```

Comment:
I don't really get this, aren't unsubscribed users represented by `active: false` - in which case this function will never return them?

Reply:
True. The idea is being able to support exceptions at the collective level, for example:
Unsubscribe from every Transaction notification but subscribe to Transaction notifications of a specific Collective.